src/pages/server/login.vue

```diff
@@ -36,10 +36,17 @@
         md="6"
         lg="5"
       >
-        <h1 v-if="!isEmpty(currentUser)" class="text-h4 mb-6 text-center">
-          {{ $t('login.loginAs', { name: currentUser.Name }) }}
-        </h1>
-        <h1 v-else class="text-h4 mb-6 text-center">{{ $t('login.login') }}</h1>
+        <v-col align="center" justify="center" dense class="mt-6 mb-3">
+          <h1 v-if="!isEmpty(currentUser)" class="text-h4 mb-3 text-center">
```

Comment:
I _think_ the 2 mb-3 are a little too much, I'd reduce them but that's up to you :)